config/schedule.yml

```diff
@@ -21,49 +21,49 @@ award_yearly_club_badges:
   class: "BadgeAchievements::BadgeAwardWorker"
```

Comment:
**potentially blocking**: will this break if there are serialized workers with the old argument after the restart deploy?

Reply:
Most of them run only every 12 hours, one only runs weekly. I don't think there should be too many queued up jobs hanging around. The only exception is `award_contributor_badges_from_github` which runs hourly at 20 minutes past the full hour, but even there we could probably miss one in the worst case and just wait for the next run to award the badges.

Reply:
Sure! We can also re-run the manually if we need to